npm-shrinkwrap.json

```diff
@@ -2,55 +2,11 @@
   "name": "dockstore-ui2",
   "version": "0.0.0",
   "dependencies": {
```

Comment:
Seems like there is no angular-cli in the shrinkwrap, so no 'ng' installed.

Reply:
Looks like it was removed in two places from the shrink-wrap, though that is weird since it was only removed from dev dependencies.

Reply:
That brings up an interesting point. For the compose_setup, I install the cli globally. https://github.com/dockstore/compose_setup/blob/master/templates/Dockerfile_ui2.template#L17 based on the readme. https://github.com/dockstore/dockstore-ui2/blob/feature/ts-node_move/README.md#bower

The .travis.yml does something different which is a mismatch